Anlab.Mvc/Views/Lab/Search.cshtml

```diff
@@ -0,0 +1,19 @@
+@{
+    ViewBag.Title = "Search";
+}
+
+
+<div class="col">
+    <form asp-controller="Lab" asp-action="Search">
+        <div class="form-horizontal">

```

Comment:
@jSylvestre I think it kinda looks funny with the word wrap of the label, I think it would look better as a regular form instead of a horizontal one